test/expected.txt

```diff
@@ -297,6 +297,11 @@
 || 
 || 0 persons listed!
 || ===================================================
+|| ===================================================
+|| Enter command: || [Command entered:  sort]
+||
+|| 0 persons listed!
+|| ===================================================
```

Comment:
Good that you have attempted to update the I/O test cases. However, your test case only checks the empty list sort scenario. Try to add more meaningful test cases in the future. Also, I would suggest adding JUnit test cases as well.